topdown/http.go

```diff
@@ -136,6 +146,32 @@ func executeHTTPRequest(bctx BuiltinContext, obj ast.Object) (ast.Value, error)
 				return nil, err
 			}
 			body = bytes.NewBuffer(bodyValBytes)
+		case "tls_use_system_certs":
+			tlsUseSystemCerts, err = strconv.ParseBool(obj.Get(val).String())
+			if err != nil {
+				return nil, err
+			}
+		case "tls_ca_cert_file":
+			tlsCaCertFile = obj.Get(val).String()
+			tlsCaCertFile = strings.Trim(tlsCaCertFile, "\"")
+		case "tls_ca_cert_env_variable":
+			caCertEnv := obj.Get(val).String()
+			caCertEnv = strings.Trim(caCertEnv, "\"")
+			tlsCaCertEnvVar = []byte(os.Getenv(caCertEnv))
```

Comment:
Do we want to check that it's non-empty? `os.Getenv` [will just return `""`](https://play.golang.org/p/Q3TJduGqr9W)